legion/legion/model/client.py

```diff
@@ -48,8 +48,8 @@ class ModelClient:
     Model HTTP client
     """
 
-    def __init__(self, model_id, model_version, token=None, host=None, http_client=None, use_relative_url=False,
-                 timeout=None):
+    def __init__(self, model_id, model_version, token=None, host=None, http_client=None, http_exception=None,
+                 use_relative_url=False, timeout=None):
         """
         Build client
```

Comment:
You missed adding docs for new arguments

Reply:
Fixed